gclient/src/api/storage/mod.rs

```diff
@@ -47,8 +47,8 @@ impl GearApi {
     /// mailbox.
     pub async fn get_from_account_mailbox(
         &self,
-        account_id: impl Borrow<AccountId32>,
-        message_id: impl Borrow<MessageId>,
+        account_id: &AccountId32,
```

Comment:
```suggestion
        account_id: impl Into<AccountId32>,
```

Reply:
Also provide From/Into AccountID <-> u64, [u8; 32], ProgramId

Reply:
So, we need to change all functions that have the `&AccountId32` argument.